WebRoot/skins/_base/base4/skin.css

```diff
@@ -1127,3 +1127,28 @@ svg.progress-circle.tick-inside {
 .ZmSignatureUsageOptions TD:last-of-type > .ZSelect {
 	padding-right: 0;
 }
+
+/*Action button container styling, matching the alignments with icons in FolderPropsDialog*/
+#FolderProperties_buttons {
+    margin-right:18px;
+}
+
+/*************
+ *
+ *	DwtGrouper: Use to show groups in dialogs like ZmSharePropsDialog & ZmFolderPropsDialog
+ *
+ *************/
+
+.DwtGrouper > fieldset {
```

Comment:
This is a behaviour that we are adding at the theme level. Basically how the field set looks like. My thought was that it can look different at skin level.